library/std/src/sys/thread_local/key/racy.rs

```diff
@@ -126,18 +76,18 @@ impl StaticKey {
     /// been allocated.
     #[inline]
     pub unsafe fn set(&self, val: *mut u8) {
-        imp::set(self.key(), val)
+        unsafe { super::set(self.key(), val) }
     }
 
     #[inline]
-    unsafe fn key(&self) -> imp::Key {
+    fn key(&self) -> super::Key {
```

Comment:
Nothing – it shouldn't have been `unsafe` to begin with, as key creation doesn't have any preconditions to uphold. Since I enabled `#[forbid(unsafe_op_in_unsafe_fn)]`, I decided to remove the `unsafe` here instead of adding a bunch of unnecessary ones elsewhere.